src/main/java/org/fundacionjala/coding/manuel/Kata5.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.manuel;
+
+
+import java.util.HashMap;
+import java.util.Map;
+/**
+ * Created by Administrator on 3/5/2018.
+ */
+public class Kata5 {
+    /**
+     * makeComplement method.
+     * @param s .
+     * @return newword .
+     */
+    public String makeComplement(String s) {
+        Map<String, String> complementaries = new HashMap<String, String>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)